src/pages/settings/Profile/Contacts/ContactMethodsPage.js

```diff
@@ -0,0 +1,138 @@
+import Str from 'expensify-common/lib/str';
+import lodashGet from 'lodash/get';
+import PropTypes from 'prop-types';
+import React, {Component} from 'react';
+import {ScrollView} from 'react-native-gesture-handler';
+import {withOnyx} from 'react-native-onyx';
+import _ from 'underscore';
+import HeaderWithCloseButton from '../../../../components/HeaderWithCloseButton';
+import ScreenWrapper from '../../../../components/ScreenWrapper';
+import withCurrentUserPersonalDetails, {withCurrentUserPersonalDetailsDefaultProps, withCurrentUserPersonalDetailsPropTypes} from '../../../../components/withCurrentUserPersonalDetails';
+import withLocalize, {withLocalizePropTypes} from '../../../../components/withLocalize';
+import CONST from '../../../../CONST';
+import compose from '../../../../libs/compose';
+import Navigation from '../../../../libs/Navigation/Navigation';
+import ONYXKEYS from '../../../../ONYXKEYS';
+import ROUTES from '../../../../ROUTES';
+import LoginField from './LoginField';
+
+const propTypes = {
+    /* Onyx Props */
+
+    /** Login list for the user that is signed in */
+    loginList: PropTypes.shape({
+        /** Value of partner name */
+        partnerName: PropTypes.string,
+
+        /** Phone/Email associated with user */
+        partnerUserID: PropTypes.string,
+
+        /** Date of when login was validated */
+        validatedDate: PropTypes.string,
+    }),
+
+    ...withLocalizePropTypes,
+    ...withCurrentUserPersonalDetailsPropTypes,
+};
+
+const defaultProps = {
+    loginList: {},
+    ...withCurrentUserPersonalDetailsDefaultProps,
+};
+
+class ContactMethodsPage extends Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            logins: this.getLogins(),
+        };
+
+        this.getLogins = this.getLogins.bind(this);
```

Comment:
Unnecessary bindings

Reply:
I see a `this.props.loginList` in `getLogins` - we don't need to bind for that?

Reply:
We don't need to bind here, we need to bind only if we pass a method as the prop to other components.

Reply:
@Beamanator bump!

Reply:
Since this is just a copy paste and the component will get redone eventually (taken from [here](https://github.com/Expensify/App/pull/15039#issuecomment-1431295856)), maybe we don't care about this type of polishing. What do you think @Santhosh-Sellavel ?

Reply:
All right then!